docs/usage/two-phase-transfer.md

```diff
@@ -5,50 +5,68 @@ protocol for distributed
 transactions](https://en.wikipedia.org/wiki/Two-phase_commit_protocol).
 
 Single-phase transfers post funds to accounts immediately when they
-are created. That is, they affect the `debits_posted` and
-`credits_posted` fields on relevant accounts.
+are created. That is, they affect the
```

Comment:
nit: s/affect/increase (to be a little more specific)

Reply:
:+1: done, thanks!